src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -230,18 +230,18 @@ def get_custom_clouds():
     return [c for c in get_clouds() if c.name not in known_cloud_names]
 
 
-def _init_known_clouds():
+def init_known_clouds(force=True):
```

Comment:
Do you mean to have the default value at `True`? Callers are not passing in `False`, so to me, this arg is not really needed.